vscode-dotnet-runtime-library/src/Utils/ExtensionConfigurationWorker.ts

```diff
@@ -2,21 +2,28 @@
 *  Licensed to the .NET Foundation under one or more agreements.
 *  The .NET Foundation licenses this file to you under the MIT license.
 *--------------------------------------------------------------------------------------------*/
-import { IExistingPath, IExtensionConfiguration } from '../IExtensionContext';
+import { IExistingPaths, IExtensionConfiguration, ILocalExistingPath } from '../IExtensionContext';
 import { IExtensionConfigurationWorker } from './IExtensionConfigurationWorker';
 
 export class ExtensionConfigurationWorker implements IExtensionConfigurationWorker {
     constructor(private readonly extensionConfiguration: IExtensionConfiguration,
                 private readonly pathConfigValueName: string | undefined) {}
 
-    public getPathConfigurationValue(): IExistingPath[] | undefined {
+    public getPathConfigurationValue(): IExistingPaths | undefined {
         return this.pathConfigValueName ? this.extensionConfiguration.get(this.pathConfigValueName) : undefined;
     }
 
-    public async setPathConfigurationValue(configValue: IExistingPath[]): Promise<void> {
+    public async setLocalPathConfigurationValue(configValue: ILocalExistingPath[]): Promise<void> {
         if (!this.pathConfigValueName) {
             throw Error('Existing path configuration not supported.');
         }
-        await this.extensionConfiguration.update<IExistingPath[]>(this.pathConfigValueName, configValue, true);
+        await this.extensionConfiguration.update<ILocalExistingPath[]>(this.pathConfigValueName, configValue, true);
+    }
+
+    public async setGlobalPathConfigurationValue(configValue: string): Promise<void> {
+        if (!this.pathConfigValueName) {
```

Comment:
`pathConfigValueName` is storing the setting for local paths. If we use that here, it will override the local path setting. We should use the string `sharedExistingDotnetPath` instead which would be stored as the globalPathKey